pkg/gateway/gateway.go

```diff
@@ -0,0 +1,84 @@
+package gateway
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/grpc-ecosystem/grpc-gateway/runtime"
+	"github.com/infobloxopen/atlas-app-toolkit/gw"
+	"google.golang.org/grpc"
+)
+
+// Option is a functional option that modifies the REST gateway on
+// initialization
+type Option func(*gateway)
+
+type registerFunc func(context.Context, *runtime.ServeMux, string, []grpc.DialOption) (err error)
+
+type gateway struct {
+	serverAddress     string
+	serverDialOptions []grpc.DialOption
+	endpoints         map[string][]registerFunc
+}
+
+// NewGateway creates a gRPC REST gateway with HTTP handlers that have been
+// generated by the gRPC gateway protoc plugin
+func NewGateway(options ...Option) (*http.ServeMux, error) {
+	// configure gateway defaults
+	g := gateway{
+		serverAddress:     ":9090",
```

Comment:
it would be great to expose this as an exported `const` instead of a string literal